dll/win32/shell32/CDefaultContextMenu.cpp

```diff
@@ -702,9 +702,8 @@ void
 CDefaultContextMenu::TryPickDefault(HMENU hMenu, UINT idCmdFirst, UINT DfltOffset, UINT uFlags)
 {
     // Are we allowed to pick a default?
-    UINT ntver = RosGetProcessEffectiveVersion();
-    if (((uFlags & CMF_NODEFAULT) && ntver >= _WIN32_WINNT_VISTA) ||
-        ((uFlags & CMF_DONOTPICKDEFAULT) && ntver >= _WIN32_WINNT_WIN7))
+    if (((uFlags & CMF_NODEFAULT)) ||
```

Comment:
The check for Vista is really not useful here? (i.e. the behaviour when the flag is set also happens on Win2k3 shell?)
```suggestion
    if ((uFlags & CMF_NODEFAULT) ||
```
(extra parentheses removed)

Reply:
Yes, I believe this is correct.

On NT5 I can't get CMF_NODEFAULT to "unpick" the default (open etc.) while I can on NT6 (this bit is not visible here, it's another check for CMF_NODEFAULT higher up in the file). But I am supposed to make it not pick a default if there was none, even on NT5.

And I kept the parens for alignment with the `||` on the next line.